src/applications/gi/components/profile/EstimatedBenefits.jsx

```diff
@@ -158,11 +158,11 @@ export const EstimatedBenefits = ({ profile, outputs, calculator }) => (
         <CalculatorResultRow
           label="Out of pocket tuition"
           value={outputs.outOfPocketTuition.value}
-          bold
+          bolds
           visible={outputs.outOfPocketTuition.visible}
         />
-      </div>
-      <div className="total-paid-to-you">
+      </ul>
+      <ul className="total-paid-to-you" aria-label="total paid to you">
```

Comment:
Let's capitalize the word "Total" and add a `role="list"`

Reply:
You got it.